app/scripts/account-import-strategies/index.js

```diff
@@ -34,9 +34,7 @@ const accountImporter = {
       let wallet
       try {
         wallet = importers.fromEtherWallet(input, password)
-      } catch (e) {
-        console.log('Attempt to import as EtherWallet format failed, trying V3...')
```

Comment:
This seems like it could be useful to know. It's probably worth keeping this message as a `log.info`, or `log.debug` at the very least.

I'm not sure if `log.info` would show up in the console output of unit tests 🤔 Probably not? `log.debug` definitely shouldn't though.

Reply:
Fair point, I've translated this to use `log`